package.json

```diff
@@ -23,8 +23,7 @@
   "author": "Jesse Gibson",
   "license": "MIT",
   "dependencies": {
-    "babel-runtime": "^6.11.6",
-    "gun": "^0.7.0"
```

Comment:
Looks like this broke [the build](https://travis-ci.org/PsychoLlama/gun-level/jobs/252397528), but it should be an easy workaround. If you add gun as a `devDependency` the tests will pass, and developers installing the package won't get it automatically. I'd recommend putting the same entry in `peerDependencies`, that way people get a warning if they're using an incompatible gun version, but it won't be bundled with the package.